.github/workflows/build_linux_packages.yml

```diff
@@ -125,6 +128,13 @@ jobs:
           python build/indexer.py -f '*.log' build/logs/
```

Comment:
The index file for logs is created in this step but the step won't be executed when the build failed. Thus this should be integrated into the log to create an index file listing all logs that were uploaded.

Reply:
Could be a separate script that teatime calls.

Reply:
I can do that -  my understanding is that this need to be called after the built is done.

Reply:
I think I need to consider the next line as well           
`sed -i 's,a href="..",a href="../../index-${{env.AMDGPU_FAMILIES}}.html",g' build/logs/index.html
`